clients/retrieval_client.go

```diff
@@ -31,6 +31,11 @@ type retrievalClient struct {
 	numConnections        int
 }
 
+type operatorFetchChunkStatusInfo struct {
+	Success             bool
```

Comment:
I don't think the success argument is actually used, so it doesn't seem like this needs a custom type.

Reply:
yes initially when i wrote i thought of tracking both retrievals that succeed from operators and that do not from operators....your comment makes sense i will remove this custom type